### PR TITLE
Add My Profile: ann-design-max

### PR DESCRIPTION
**Pull request by Ann-design-max**
Please add my profile: ann-design-max 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1730"></a>

### CODE REVIEW

**Review by eddiejaoude — APPROVED** [review]
Looks god 👍

File: public/data/ann-design-max.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "name": "Anastestia Onyekaba",
+  "type": "personal",
+  "bio": "UI/UX Designer | Frontend Developer | Co-Founder of N&S Tech | Open Source Enthusiast!",
+  "avatar": "https://github.com/Ann-design-max.png",
+  "links": [
+    {
+      "name": "Follow me on Github ",
```
Comment (Panquesito7) [inline public/data/ann-design-max.json:8]: I recommend capitalizing the `h` here. What do you think? 😄
```suggestion
      "name": "Follow me on GitHub",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Review by vedanshi555 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Ann-design-max** [comment]
> Looks good 👍

thanks @eddiejaoude . And thank you on behalf of the millions you are helping out ❤️

File: public/data/ann-design-max.json (hunk 1 above)
Comment (Ann-design-max) [reply public/data/ann-design-max.json]: @Panquesito7 Thank you for pointing this out. It has been resolved in this commit https://github.com/EddieHubCommunity/LinkFree/pull/1730/commits/6507b212b6485bd8b4be8654194b1acbc3ebdf51

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**